### ISSUE TITLE
mitre_table_info table doesn't exist in clickhouse cluster

### ISSUE THREAD

**Issue by mcodato** [ntop/ntopng#8636]
**Environment**:
* OS: rhel 8.10
* Architecture: x86_64
* ntopng version/revision: 6.2.240813

**What happened**:
The logs report that ntopng cannot insert into **mitre_table_info** because the mitre_table_info table does not exist.

It apparently appears that the schema used in a [cluster environment](https://github.com/ntop/ntopng/blob/dev/httpdocs/misc/db_schema_clickhouse_cluster.sql) is different than the schema used in a [non-cluster environment](https://github.com/ntop/ntopng/blob/dev/httpdocs/misc/db_schema_clickhouse.sql)

**How did you reproduce it?**
- Setup clickhouse as cluster 
- Run ntopng

**Debug Information**:

```
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [startup.lua:202] Running /usr/share/ntopng/local/scripts/callbacks/system/startup.lua
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [startup.lua:210] Importing ClickHouse dumps...
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:756] Disconnected from MySQL for interface __system__...
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:784] Enabled TLS support in MySQL client connection
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:949] Successfully connected to ClickHouse [ntopng@clickhouse-server.neteyelocal:9004][dbname: ntopng] for interface __system__
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:45] ERROR: SQL query: INSERT INTO mitre_table_info (ALERT_ID, ENTITY_ID, TACTIC, TECHNIQUE, SUB_TECHNIQUE, MITRE_ID) VALUES (45,4,3,1505,0,'T1505'),(10,1,10,1048,0,'T1048'),(6,4,10,1048,0,'T1048'),(2,1,11,1071,107104,'T1071.004'),(33,4,11,1071,107104,'T1071.004'),(66,4,6,1557,0,'T1557'),(28,4,6,1557,0,'T1557'),(83,4,6,1557,0,'T1557'),(52,4,6,1187,0,'T1187'),(98,4,43,1598,156603,'T1598.001'),(62,4,0,1602,0,'T1003'),(4183,0,6,1040,0,'T1040'),(85,4,7,1046,0,'T1018'),(81,4,7,1046,0,'T1018'),(39,4,6,1539,0,'T1539'),(16,4,8,1563,156302,'T1563.002'),(42,4,11,1071,0,'T1071.001'),(27,4,40,1496,0,'T1496'),(25,4,11,1573,0,'T1573'),(43,4,5,1070,0,'T1070'),(76,4,5,1070,0,'T1070'),(23,4,5,1070,0,'T1070'),(74,4,40,1565,0,'T1565'),(55,4,40,1565,0,'T1565'),(95,4,40,1565,0,'T1565'),(37,4,40,1565,0,'T1565'),(93,4,40,1565,0,'T1565'),(94,4,40,1565,0,'T1565'),(82,4,11,1001,100103,'T1001.003'),(46,4,11,1001,100103,'T1001.003'),(11,4,9,1039,0,'T1039'),(4,1,40,1498,0,'T1498'),(26,4,11,1102,110203,'T1102.003'),(32,4,5,1207,0,'T1207'),(78,4,8,1570,0,'T1570'),(49,4,2,0,0,'T1203'),(24,4,2,0,0,'T1203'),(70,4,1,1566,156602,'T1566.002'),(30,4,11,0,0,'T1571'),(22,4,1,1078,0,'T1078'),(69,4,1,1078,0,'T1078'),(44,4,1,1189,0,'T1189'),(68,4,1,1190,0,'T1190'),(40,4,1,1190,0,'T1190'),(8,4,9,1602,0,'T1602.002'),(65,4,43,1596,159601,'T1596.001'),(64,4,43,1596,159601,'T1596.001'),(86,4,1,1566,0,'T1566'),(1,1,11,1071,107103,'T1071.003'),(34,4,11,1071,107103,'T1071.003'),(61,4,8,1021,109804,'T1021.004'),(50,4,8,1021,102102,'T1021.002'),(54,4,11,1090,0,'T1090'),(5,1,43,1595,0,'T1595'),(21,1,43,1595,0,'T1595'),(20,1,43,1595,0,'T1595'),(20,1,43,1595,0,'T1595'),(25,1,43,1595,0,'T1595'),(53,4,40,1496,0,'T1496.001'),(38,4,8,1021,102101,'T1021.001'),(20,4,7,1046,0,'T1046'),(56,4,7,1046,0,'T1046'),(67,4,7,1046,0,'T1046'),(7,1,11,1090,109002,'T1090.002'),(75,4,3,1205,120501,'T1205.001'),(47,4,11,1568,156802,'T1568.002'),(12,1,3,1133,0,'T1133'),(29,1,3,1133,0,'T1133'),(57,4,3,1133,0,'T1133'),(58,4,8,0,0,'TA0008'),(84,4,1,0,0,'T1659'),(79,4,1,0,0,'T1659'),(60,4,5,1562,0,'T1562'),(16,1,11,1102,0,'T1102'),(29,4,2,1204,120401,'T1204.001'),(14,1,10,1041,0,'T1041'),(88,4,10,1041,0,'T1029'),(59,4,10,1041,0,'T1029'),(5,4,11,1071,0,'T1071'),(41,4,2,0,0,'TA0002'),(48,4,3,1205,0,'T1205'),(100,4,40,1499,0,'T1499'),(36,4,40,1499,0,'T1499'),(12,4,40,1499,0,'T1499'),(12,4,40,1499,0,'T1499'),(71,4,40,1499,0,'T1499'),(7,4,11,1568,156803,'T1568.003'),(72,4,43,1596,159503,'T1595.003'),(6,1,40,1498,149801,'T1498.001'),(18,1,40,1498,149801,'T1498.001'),(23,1,40,1498,149801,'T1498.001'),(51,4,11,1090,109004,'T1090.004'),(3,1,40,1498,149802,'T1498.002'),(22,1,40,1498,149802,'T1498.002'),(35,4,40,1498,149802,'T1498.002'),(73,4,5,1027,0,'T1027'),(63,4,5,1027,0,'T1027');
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:45] ERROR: SQL query: INSERT INTO mitre_table_info (ALERT_ID, ENTITY_ID, TACTIC, TECHNIQUE, SUB_TECHNIQUE, MITRE_ID) VALUES (45,4,3,1505,0,'T1505'),(10,1,10,1048,0,'T1048'),(6,4,10,1048,0,'T1048'),(2,1,11,1071,107104,'T1071.004'),(33,4,11,1071,107104,'T1071.004'),(66,4,6,1557,0,'T1557'),(28,4,6,1557,0,'T1557'),(83,4,6,1557,0,'T1557'),(52,4,6,1187,0,'T1187'),(98,4,43,1598,156603,'T1598.001'),(62,4,0,1602,0,'T1003'),(4183,0,6,1040,0,'T1040'),(85,4,7,1046,0,'T1018'),(81,4,7,1046,0,'T1018'),(39,4,6,1539,0,'T1539'),(16,4,8,1563,156302,'T1563.002'),(42,4,11,1071,0,'T1071.001'),(27,4,40,1496,0,'T1496'),(25,4,11,1573,0,'T1573'),(43,4,5,1070,0,'T1070'),(76,4,5,1070,0,'T1070'),(23,4,5,1070,0,'T1070'),(74,4,40,1565,0,'T1565'),(55,4,40,1565,0,'T1565'),(95,4,40,1565,0,'T1565'),(37,4,40,1565,0,'T1565'),(93,4,40,1565,0,'T1565'),(94,4,40,1565,0,'T1565'),(82,4,11,1001,100103,'T1001.003'),(46,4,11,1001,100103,'T1001.003'),(11,4,9,1039,0,'T1039'),(4,1,40,1498,0,'T1498'),(26,4,11,1102,110203,'T1102.003'),(32,4,5,1207,0,'T1207'),(78,4,8,1570,0,'T1570'),(49,4,2,0,0,'T1203'),(24,4,2,0,0,'T1203'),(70,4,1,1566,156602,'T1566.002'),(30,4,11,0,0,'T1571'),(22,4,1,1078,0,'T1078'),(69,4,1,1078,0,'T1078'),(44,4,1,1189,0,'T1189'),(68,4,1,1190,0,'T1190'),(40,4,1,1190,0,'T1190'),(8,4,9,1602,0,'T1602.002'),(65,4,43,1596,159601,'T1596.001'),(64,4,43,1596,159601,'T1596.001'),(86,4,1,1566,0,'T1566'),(1,1,11,1071,107103,'T1071.003'),(34,4,11,1071,107103,'T1071.003'),(61,4,8,1021,109804,'T1021.004'),(50,4,8,1021,102102,'T1021.002'),(54,4,11,1090,0,'T1090'),(5,1,43,1595,0,'T1595'),(21,1,43,1595,0,'T1595'),(20,1,43,1595,0,'T1595'),(20,1,43,1595,0,'T1595'),(25,1,43,1595,0,'T1595'),(53,4,40,1496,0,'T1496.001'),(38,4,8,1021,102101,'T1021.001'),(20,4,7,1046,0,'T1046'),(56,4,7,1046,0,'T1046'),(67,4,7,1046,0,'T1046'),(7,1,11,1090,109002,'T1090.002'),(75,4,3,1205,120501,'T1205.001'),(47,4,11,1568,156802,'T1568.002'),(12,1,3,1133,0,'T1133'),(29,1,3,1133,0,'T1133'),(57,4,3,1133,0,'T1133'),(58,4,8,0,0,'TA0008'),(84,4,1,0,0,'T1659'),(79,4,1,0,0,'T1659'),(60,4,5,1562,0,'T1562'),(16,1,11,1102,0,'T1102'),(29,4,2,1204,120401,'T1204.001'),(14,1,10,1041,0,'T1041'),(88,4,10,1041,0,'T1029'),(59,4,10,1041,0,'T1029'),(5,4,11,1071,0,'T1071'),(41,4,2,0,0,'TA0002'),(48,4,3,1205,0,'T1205'),(100,4,40,1499,0,'T1499'),(36,4,40,1499,0,'T1499'),(12,4,40,1499,0,'T1499'),(12,4,40,1499,0,'T1499'),(71,4,40,1499,0,'T1499'),(7,4,11,1568,156803,'T1568.003'),(72,4,43,1596,159503,'T1595.003'),(6,1,40,1498,149801,'T1498.001'),(18,1,40,1498,149801,'T1498.001'),(23,1,40,1498,149801,'T1498.001'),(51,4,11,1090,109004,'T1090.004'),(3,1,40,1498,149802,'T1498.002'),(22,1,40,1498,149802,'T1498.002'),(35,4,40,1498,149802,'T1498.002'),(73,4,5,1027,0,'T1027'),(63,4,5,1027,0,'T1027');
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:46] ERROR: SQL error (1): Code: 60. DB::Exception: Table ntopng.mitre_table_info doesn't exist. (UNKNOWN_TABLE) (version 22.9.2.7 (official build))
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:46] ERROR: SQL error (1): Code: 60. DB::Exception: Table ntopng.mitre_table_info doesn't exist. (UNKNOWN_TABLE) (version 22.9.2.7 (official build))
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:756] Disconnected from MySQL for interface __system__...
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:784] Enabled TLS support in MySQL client connection
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:949] Successfully connected to ClickHouse [ntopng@clickhouse-server.neteyelocal:9004][dbname: ntopng] for interface __system__
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:45] ERROR: SQL query: OPTIMIZE TABLE mitre_table_info FINAL;
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:45] ERROR: SQL query: OPTIMIZE TABLE mitre_table_info FINAL;
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:46] ERROR: SQL error (1): Code: 60. DB::Exception: Table ntopng.mitre_table_info doesn't exist. (UNKNOWN_TABLE) (version 22.9.2.7 (official build))
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [MySQLDB.cpp:46] ERROR: SQL error (1): Code: 60. DB::Exception: Table ntopng.mitre_table_info doesn't exist. (UNKNOWN_TABLE) (version 22.9.2.7 (official build))
Aug 19 06:34:37 nemc ntopng[120406]: 19/Aug/2024 06:34:37 [startup.lua:253] Completed startup.lua
```

**Comment by cardigliano** [comment]
Fixed in 5768dd88d6ce510e7fe7070517b1a07ca9695701, an update will be available later today

**Comment by mcodato** [comment]
It works, thank you very much
